tests/cover/test_phases.py

```diff
@@ -84,7 +84,7 @@ def test_usage(i):
     with pytest.raises(ValueError):
         test_usage()
 
-    saved, = database.data.values()
+    saved, = [v for k, v in database.data.items() if b'coverage' not in k]
```

Comment:
Please just use `len(non_covering_examples(database))` here, as for the other tests.